migrations/6_update_dates_existing_published_projects.js

```diff
@@ -0,0 +1,48 @@
+'use strict';
+
+exports.up = function (knex, Promise) {
+  return knex('publishedProjects').select('id', 'date_created', 'date_updated')
```

Comment:
You should add a `WHERE` clause here to filter out projects that already have a date_created and date_updated value. Something like `WHERE date_created IS NOT NULL AND date_updated IS NOT NULL`